environment.yml

```diff
@@ -16,17 +14,18 @@ dependencies:
   - fsspec
   - geopandas
   - h5netcdf
-  - intake
-  - intake-esm >=2022.9.18
+  - h5py
+  - intake-esm >=2023.07.07
   - matplotlib
   - netCDF4
   - numpy
   - pandas >= 2
   - parse
-  - pygeos
```

Comment:
On utilise `pygeos` ! https://github.com/Ouranosinc/xscen/blob/94fca2e6190d0e7524d3d1678e46faf0fca51c3c/xscen/aggregate.py#L500

Reply:
I think the confusion is because pygeos comes bundled with Shapely now (IIRC). But given that we _explicitly_ call it, we should also explicitly mark it in the recipe(s).

Reply:
Pareil pour:
```python3
import xscen as xs

ds = xr.tutorial.open_dataset("air_temperature")
xs.aggregate.spatial_mean(ds, method='cos-lat')
```

Je suis sur le projet.

Reply:
@Zeitsperre I think that it's rather that `pygeos` has been _integrated_ into `shapely`. The package still exist, but is not needed anymore, the functions are effectively migrated to the shapely API. In fact, having both `pygeos` and `shapely >= 2` triggers a warning in `geopandas` because of the "ambiguity". 

In theory, replacing `pygeos` by `shapely` works directly. However, this made me realize that the function is broken, so...